spec/features/user_features/user_views_spec.rb

```diff
@@ -22,4 +22,26 @@
     expect(page).to have_content "Email"
     expect(page).to have_content "Password"
   end
+
+  scenario "if user is signed in, should not see log in or sign up" do
+    user = User.create(email: "omar@gmail.com", password: "123456")
+    # sign_in user
+    visit new_user_session_path
+    fill_in "Email", with: "omar@gmail.com"
+    fill_in "Password", with: "123456"
+    click_button "Log in"
+    expect(page).to have_content "Log Out"
+    expect(page).to have_content "Log Out"
+    expect(page).not_to have_content "Log In"
+    expect(page).not_to have_content "Sign Up"
+  end
+
+  scenario "if user is not signed in, should not see log out" do
+    visit root_path
+    expect(page).to have_content "Log In"
+    expect(page).to have_content "Sign Up"
+    expect(page).not_to have_content "Log Out"
+  end
+
+
```

Comment:
I am not wild about the name of this file. `user_views_spec`? User views what, exactly?